jovian/cli.py

```diff
@@ -1,10 +1,11 @@
 import argparse
-import webbrowser
+# import webbrowser
 import os
 
 from jovian.utils.clone import clone, pull
 from jovian.utils.install import install, activate
 from jovian._version import __version__
+from jovian.utils.api import add_slack
```

Comment:
this will probably cause the CLI to crash on Python 2.7 , because the `jovian.utils.api` module uses some features from Python 3. 

Anything that is included as a CLI command needs to be dealt with separately. See `jovian.utils.clone` for an example.